### ISSUE TITLE
Data Engineer (DevOps) (#2035710)

### ISSUE THREAD

**Issue by PaigeMcAlister19** [tech256/jobs#1061]
PeopleTec is seeking a Data Engineer (DevOps) to support our Huntsville, AL location.

We are looking for a motivated self-starter to support our customers by contributing to research concepts, technical investigative studies, lab demonstrations and resolve or develop solutions to hard problems as they relate to Cybersecurity, Artificial Intelligence/Machine Learning, Modeling and Simulation, Test and Evaluation initiatives. The candidate will assist Project Managers, Product Owners, and other team members to meet the Customer's mission objectives.

In this role you will:

Support multiple unique projects occurring at the same time and frequently articulate progress and challenges to the Task Leads

Use a variety of algorithmic methods, using a variety of data tools, building, and implementing models and creating/running training data

Manipulate data sets and building statistical models

Drive to learn and master new technologies and techniques

Work with a diverse team of Cyber Engineers, Scientists, Analysts and Developers

Required Skills:

Must have thorough understanding and experience in the following:

Python, C++, C#, Visual Basic

Git, VSCode, Visual Studio, Jupyter Notebook, MATLAB, AutoCAD, BRLCAD, Solid Edge, Microsoft Office Suite

Strong problem-solving skills with an emphasis on supervised machine learning

Aid in creation of testing practices and DevOps for software using GitLab

Mine and analyze data from network logs to drive improvement of operating methods, anomaly and event detection, and network defense strategies

Working with and creating data architectures

Adept at using large data sets to find opportunities for machine learning and using models to test the effectiveness of different courses of action

Passion for discovering solutions hidden in large data sets and working with stakeholders to improve decision making

Travel: up to 10 % during safe travel conditions

Must be a U.S. Citizen

An active DoD Secret clearance is required to perform this work. Candidates are required to have an active Secret clearance upon hire, and the ability to maintain this level of clearance during their employment.

-

Education Requirements:

Bachelor's degree in Engineering, Mathematics, Computer Science or another quantitative field with 3 years of related experience or master's degree in Engineering, Mathematics, Computer Science or another quantitative field with 1 year of related experience.

-

Desired Skills:

Coding experience with several languages to include querying databases

Analyzing data from 3rd party providers such as Microsoft Azure and Google APIs

Evaluate the effectiveness of adversarial models used to attack existing machine learning algorithms

Assess the effectiveness and accuracy of new data sources and industry-leading learning techniques

Develop custom data models and algorithms to apply to data sets

Use predictive modeling to increase and optimize operational outcomes

Experience using web services: Redshift, S3, Spark, etc.

Experience using log file analyzers and visualization tools such as Splunk, Elasticsearch, Logstash, and Kibana (ELK), both on premises and in the cloud

Experience using statistical computer languages (R, Python, SQL, etc.) to manipulate data and draw insights from large data sets

Knowledge and experience in advanced statistical and data mining techniques (GLM/Regression, Random Forest, Boosting, Trees, text mining, social network analysis, etc.)

Experience with applications that serve inferences from trained models

Experience creating and using advanced machine learning algorithms and techniques (simulation, scenario analysis, clustering, decision tree learning, artificial neural networks, etc.) and their real-world advantages/drawbacks

Experience with distributed data/computing tools to include: Map/Reduce, Hadoop, Hive, Spark, SQL Server, etc.

Experience with technical writing (technical papers, online forums, articles)

People First. Technology Always.

PeopleTec, Inc. is an employee-owned small business founded in Huntsville, AL that provides exceptional customer support by employing and retaining a highly skilled workforce.

Culture: The name "PeopleTec" was deliberately chosen to remind us of our core value system - our people. Our company's foundation was built on placing our employees and customers first. With an award-winning atmosphere, we have matured into a company that boasts the best and brightest across multiple technical fields.

Career: At PeopleTec, we value your long-term goals. Whether it's through our continuing-education opportunities, our robust training programs, or our "People First" benefits package, PeopleTec truly believes that our best investments are our people.

Come Experience It.

#cjpost #dpost

EOE/Minorities/Females/Veterans/Disabled

PeopleTec, Inc. is an Equal Employment Opportunity employer and provides reasonable accommodation for qualified individuals with disabilities and disabled veterans in its job application procedures. If you have any difficulty using our online system and you need an accommodation due to a disability, you may use the following email address, HR@peopletec.com and/or phone number (256.319.3800) to contact us about your interest in employment with PeopleTec, Inc.

All qualified applicants will receive consideration for employment without regard to race, color, religion, sex, sexual orientation, gender identity, national origin, age, genetic information, citizenship, ancestry, marital status, protected veteran status, disability status or any other status protected by federal, state, or local law. PeopleTec, Inc. participates in E-Verify.

For more information, or to apply now, you must go to the website below. Please DO NOT email your resume to us as we only accept applications through our website.

https://www.applicantpro.com/j/2035710-424912
